schema/vc.graphqls

```diff
@@ -0,0 +1,102 @@
+extend type Query {
+  """
+  vinVCLatest returns the latest VINVC data for a given token.
+
+  Required Privileges: [VEHICLE_VIN_CREDENTIAL]
+  """
+  vinVCLatest(
+    """
+    The token ID of the vehicle.
+    """
+    tokenId: Int!
+  ): VINVC
+    @requiresVehicleToken
+    @requiresPrivileges(privileges: [VEHICLE_VIN_CREDENTIAL])
+
+  """
+  pomVCLatest returns the latest POMVC data for a given token.
+
+  Required Privileges: [VEHICLE_ALL_TIME_LOCATION]
+  """
+  pomVCLatest(
+    """
+    The token ID of the vehicle.
+    """
+    tokenId: Int!
+  ): POMVC
+    @requiresVehicleToken
+    @requiresPrivileges(privileges: [VEHICLE_ALL_TIME_LOCATION])
+}
+
+type VINVC {
+  """
+  vehicleTokenId is the token ID of the vehicle.
+  """
+  vehicleTokenId: Int
```

Comment:
I think GraphQL is kinda down on required fields in general but is there any universe where we don't have all of these besides `countryCode`?